src/main/java/com/google/api/codegen/discovery/transformer/go/GoSampleMethodToViewTransformer.java

```diff
@@ -96,7 +96,10 @@ private SampleView createSampleView(SampleTransformerContext context) {
       fieldVarNames.add(requestBodyVarName);
     }
 
-    if (methodInfo.isPageStreaming()) {
+    // The Go client only considers methods whose verb is "GET" to be page
```

Comment:
This appears to be changed in https://code-review.googlesource.com/#/c/10110/3/google-api-go-generator/gen.go . For reference, here's a list of non-GET method IDs which we interpret to be paging for at least one API version:

resourceviews.regionViews.listresources                                                                                            
compute.instanceGroups.listInstances                                                                                               
genomics.referencesets.search                                                                                                      
genomics.variants.search                                                                                                           
compute.regionInstanceGroups.listInstances                                                                                         
logging.entries.list                                                                                                               
resourceviews.zoneViews.listresources                                                                                              
identitytoolkit.relyingparty.downloadAccount                                                                                       
genomics.annotations.search                                                                                                        
genomics.annotationsets.search                                                                                                     
cloudresourcemanager.organizations.search                                                                                          
genomics.variantsets.search                                                                                                        
genomics.reads.search                                                                                                              
genomics.annotationSets.search                                                                                                     
dfareporting.dimensionValues.query                                                                                                 
genomics.readgroupsets.search                                                                                                      
genomics.jobs.search                                                                                                               
cih.userInteractions.list                                                                                                          
genomics.callsets.search                                                                                                           
genomics.references.search